security/certificate_types_descriptions/aggregated-api-server-certificates.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="cert-types-aggregated-api-server-certificates"]
+= Aggregated API server certificates
```

Comment:
It seems that you're describing Kubernetes API Aggregation client certificates

Reply:
I think this matches the title here https://github.com/openshift/api/blob/master/tls/docs/Aggregated%20API%20Server%20Certificates/README.md so maybe being consistent just keep it set to that.

Reply:
To stay consistent, we should name them the same, and the name should be correct. You can name both the sections `Aggregated API client certificates`.

Reply:
made that correction and will update the title in the other file https://github.com/openshift/api/blob/master/tls/docs/Aggregated%20API%20Server%20Certificates/README.md